lib/cli/kit/levenshtein.rb

```diff
@@ -67,6 +71,7 @@ def distance(str1, str2)
       # faster than `[a, b, c].min` and puts less GC pressure.
       # See https://github.com/yuki24/did_you_mean/pull/1 for a performance
       # benchmark.
+      sig { params(a: Integer, b: Integer, c: Integer).returns(Integer) }
```

Comment:
Just for this kind of use-cases Sorbet should accept `params(a, b, c: Integer)` and apply the `Integer` to all three parameters.